src/System.Private.CoreLib/shared/System/Globalization/RegionInfo.cs

```diff
@@ -50,6 +48,8 @@ public RegionInfo(string name)
             }
 
             SetName(name);
+            // TODO-NULLABLE: workaround compiler bug
+            _name = name!;
```

Comment:
Ditto; please avoid making changes that impact the generated IL just to satisfy the compiler warnings.

Reply:
changed to  `= null!` - compiler team will no-op these